Cargo.toml

```diff
@@ -7,6 +7,7 @@ license = "MIT/Apache-2.0"
 repository = "https://github.com/sfackler/log4rs"
 readme = "README.md"
 keywords = ["logger", "log", "logging"]
+autoexamples = false
```

Comment:
Based on my understanding of autoexamples, we shouldnt need this since cargo should merge targets found with those specified witn `[[example]]`

Reply:
@estk Oh true. I'll remove this.